src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    public static final int LENGTH_DEADLINE = 9;
+    public static final int LENGTH_EVENT = 6;
+    public static final int LENGTH_TODO = 5;
+    public static final int SIZE_DONE_COMMAND = 2;
+
+    public static void getDateTime() {
+        LocalDateTime myDateObj = LocalDateTime.now();
+        DateTimeFormatter myFormatObj = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+        String formattedDate = myDateObj.format(myFormatObj);
+        System.out.println(formattedDate);
+    }
+
+    public static void completeTask(ArrayList<Task> tasks, int taskIndex) {
+        taskIndex--; // index starts from 0, unlike listing number
+        if ( (taskIndex < tasks.size()) || (taskIndex > 0)) { // check if out of bounce
+            Task currentTask = tasks.get(taskIndex);
+            if (currentTask.getStatus()) { // check if already completed
+                System.out.println("Task already completed!\n");
+            } else {
+                currentTask.markAsDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println( "["+ currentTask.getTaskType() + "][" + currentTask.getStatusIcon() + "] " + currentTask.getDescription() + "\n");
```

Comment:
This line is a little bit too long. Maybe you can split this line into two rows.